modular_skyrat/modules/implants/code/augments_head.dm

```diff
@@ -140,6 +140,7 @@
 		/obj/machinery/computer/communications,
 		/obj/machinery/computer/arcade,
 		/obj/machinery/computer/holodeck,
+		/obj/machinery/computer/emergency_shuttle,
```

Comment:
This still needs the recycler blacklisted. Giving anybody access to gib people thrown in disposals is going to be an administrative nightmare.

Reply:
Emagged recyclers don't gib people

Reply:
Isn't that the entire point of emagging them, to turn off their safeties?

Reply:
> Emagging the recycler doesn't make it gib people, just makes it instakill anyone who passes through it and instantly deletes every single item a person is wearing/carrying with no way to recover them.

That sounds equally bad if it deletes everything on a person

Reply:
You still should not be able to emag recyclers with this, they're VERY destructive, I'd rather we don't have to worry about people using it for that.